\name{ GSE18520_eset }
\alias{ GSE18520_eset }
\docType{data}
\title{ A gene signature predictive for outcome in advanced ovarian cancer identifies a survival factor: microfibril-associated glycoprotein 2. }
\description{ Advanced stage papillary serous tumors of the ovary are responsible for the majority of ovarian cancer deaths, yet the molecular determinants modulating patient survival are poorly characterized. Here, we identify and validate a prognostic gene expression signature correlating with survival in a series of microdissected serous ovarian tumors. Independent evaluation confirmed the association of a prognostic gene microfibril-associated glycoprotein 2 (MAGP2) with poor prognosis, whereas in vitro mechanistic analyses demonstrated its ability to prolong tumor cell survival and stimulate endothelial cell motility and survival via the alpha(V)beta(3) integrin receptor. Increased MAGP2 expression correlated with microvessel density suggesting a proangiogenic role in vivo. Thus, MAGP2 may serve as a survival-associated target. }
\usage{data( GSE18520_eset )}
\format{
\preformatted{
experimentData(eset):
Experiment data
  Experimenter name: Mok SC, Bonome T, Vathipadiekal V, Bell A, Johnson ME, Wong KK, Park DC, Hao K, Yip DK, Donninger H, Ozbun L, Samimi G, Brady J, Randonovich M, Pise-Masison CA, Barrett JC, Wong WH, Welch WR, Berkowitz RS, Birrer MJ.A gene signature predictive for outcome in advanced ovarian cancer identifies a survival factor: microfibril-associated glycoprotein 2. Cancer Cell. 2009 Dec 8; 16(6):521-32. 
  Laboratory: Mok, Birrer 2009 
  Contact information:  
  Title: A gene signature predictive for outcome in advanced ovarian cancer identifies a survival factor: microfibril-associated glycoprotein 2. 
  URL:  
  PMIDs: 19962670 

  Abstract: A 110 word abstract is available. Use 'abstract' method.
  Information is available on: preprocessing 
  notes:
   platform_title:     
      [HG-U133_Plus_2] Affymetrix Human Genome U133 Plus 2.0 Array
   platform_shorttitle:     
      Affymetrix HG-U133Plus2
   platform_summary:     
      hgu133plus2
   platform_manufacturer:     
      Affymetrix|Operon
   platform_distribution:     
      commercial|non-commercial
   platform_accession:     
      GPL570|GPL9216
   version:     
      2015-09-22 19:21:25

featureData(eset):
An object of class 'AnnotatedDataFrame'
  featureNames: 1007_s_at 1053_at ... AFFX-HUMISGF3A/M97935_MB_at
    (42447 total)
  varLabels: probeset gene EntrezGene.ID best_probe
  varMetadata: labelDescription

}}
\details{
\preformatted{
assayData: 42447 features, 63 samples
Platform type: 
Overall survival time-to-event summary (in years):
Call: survfit(formula = Surv(time, cens) ~ -1)

   10 observations deleted due to missingness 
      n  events  median 0.95LCL 0.95UCL 
  53.00   41.00    2.05    1.48    3.70 

--------------------------- 
Available sample meta-data: 
--------------------------- 

alt_sample_name: 
   Min. 1st Qu.  Median    Mean 3rd Qu.    Max. 
  312.0   395.0   694.0   893.3  1040.0  2237.0 

sample_type: 
healthy   tumor 
     10      53 

histological_type: 
 ser NA's 
  53   10 

primarysite: 
ov 
63 

summarygrade: 
high NA's 
  53   10 

summarystage: 
late NA's 
  53   10 

tumorstage: 
   3 NA's 
  53   10 

grade: 
   3 NA's 
  53   10 

days_to_death: 
   Min. 1st Qu.  Median    Mean 3rd Qu.    Max.    NA's 
    150     450     630    1212    1440    4500      10 

vital_status: 
deceased   living     NA's 
      41       12       10 

debulking: 
optimal 
     63 

percent_normal_cells: 
 0 
63 

percent_stromal_cells: 
 0 
63 

percent_tumor_cells: 
100 
 63 

batch: 
2004-03-12 2004-04-08 2004-04-09 2004-07-20 2004-08-12 2004-08-13 2004-09-30 
        20          6          9         11         10          1          6 

uncurated_author_metadata: 
                                                title: Normal Ovary, 2008///geo_accession: GSM462643///status: Public on Oct 17 2009///submission_date: Oct 16 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: normal ovarian surface epithelium (OSE)///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian surface epithelium (OSE)///characteristics_ch1.1: ///characteristics_ch1.2: ///characteristics_ch1.3: ///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from normal ovary.///description.1: HOSE2008///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM462nnn/GSM462643/GSM462643.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                                                title: Normal Ovary, 2061///geo_accession: GSM462644///status: Public on Oct 17 2009///submission_date: Oct 16 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: normal ovarian surface epithelium (OSE)///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian surface epithelium (OSE)///characteristics_ch1.1: ///characteristics_ch1.2: ///characteristics_ch1.3: ///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from normal ovary.///description.1: HOSE2061///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM462nnn/GSM462644/GSM462644.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                                                title: Normal Ovary, 2064///geo_accession: GSM462645///status: Public on Oct 17 2009///submission_date: Oct 16 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: normal ovarian surface epithelium (OSE)///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian surface epithelium (OSE)///characteristics_ch1.1: ///characteristics_ch1.2: ///characteristics_ch1.3: ///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from normal ovary.///description.1: HOSE2064///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM462nnn/GSM462645/GSM462645.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                                                title: Normal Ovary, 2085///geo_accession: GSM462646///status: Public on Oct 17 2009///submission_date: Oct 16 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: normal ovarian surface epithelium (OSE)///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian surface epithelium (OSE)///characteristics_ch1.1: ///characteristics_ch1.2: ///characteristics_ch1.3: ///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from normal ovary.///description.1: HOSE2085///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM462nnn/GSM462646/GSM462646.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                                                title: Normal Ovary, 2225///geo_accession: GSM462647///status: Public on Oct 17 2009///submission_date: Oct 16 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: normal ovarian surface epithelium (OSE)///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian surface epithelium (OSE)///characteristics_ch1.1: ///characteristics_ch1.2: ///characteristics_ch1.3: ///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from normal ovary.///description.1: HOSE2225///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM462nnn/GSM462647/GSM462647.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                                                title: Normal Ovary, 2226///geo_accession: GSM462648///status: Public on Oct 17 2009///submission_date: Oct 16 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: normal ovarian surface epithelium (OSE)///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian surface epithelium (OSE)///characteristics_ch1.1: ///characteristics_ch1.2: ///characteristics_ch1.3: ///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from normal ovary.///description.1: HOSE2226///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM462nnn/GSM462648/GSM462648.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                                                title: Normal Ovary, 2228///geo_accession: GSM462649///status: Public on Oct 17 2009///submission_date: Oct 16 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: normal ovarian surface epithelium (OSE)///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian surface epithelium (OSE)///characteristics_ch1.1: ///characteristics_ch1.2: ///characteristics_ch1.3: ///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from normal ovary.///description.1: HOSE2228///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM462nnn/GSM462649/GSM462649.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                                                title: Normal Ovary, 2230///geo_accession: GSM462650///status: Public on Oct 17 2009///submission_date: Oct 16 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: normal ovarian surface epithelium (OSE)///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian surface epithelium (OSE)///characteristics_ch1.1: ///characteristics_ch1.2: ///characteristics_ch1.3: ///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from normal ovary.///description.1: HOSE2230///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM462nnn/GSM462650/GSM462650.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                                                title: Normal Ovary, 2234///geo_accession: GSM462651///status: Public on Oct 17 2009///submission_date: Oct 16 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: normal ovarian surface epithelium (OSE)///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian surface epithelium (OSE)///characteristics_ch1.1: ///characteristics_ch1.2: ///characteristics_ch1.3: ///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from normal ovary.///description.1: HOSE2234///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM462nnn/GSM462651/GSM462651.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                                                title: Normal Ovary, 2237///geo_accession: GSM462652///status: Public on Oct 17 2009///submission_date: Oct 16 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: normal ovarian surface epithelium (OSE)///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian surface epithelium (OSE)///characteristics_ch1.1: ///characteristics_ch1.2: ///characteristics_ch1.3: ///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from normal ovary.///description.1: HOSE2237///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM462nnn/GSM462652/GSM462652.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
title: Ovarian Tumor, 1109///geo_accession: GSM461390///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 34 (A)///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 1109///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461390/GSM461390.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
    title: Ovarian Tumor, 1214///geo_accession: GSM461391///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 17///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 1214///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461391/GSM461391.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
    title: Ovarian Tumor, 1231///geo_accession: GSM461367///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 15///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 1231///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461367/GSM461367.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
title: Ovarian Tumor, 1562///geo_accession: GSM461368///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 19 (A)///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 1562///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461368/GSM461368.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
title: Ovarian Tumor, 1660///geo_accession: GSM461369///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 15 (A)///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 1660///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461369/GSM461369.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
title: Ovarian Tumor, 1993///geo_accession: GSM461400///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 11 (A)///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 1993///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461400/GSM461400.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
      title: Ovarian Tumor, 312///geo_accession: GSM461379///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 48///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 312///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461379/GSM461379.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
 title: Ovarian Tumor, 317///geo_accession: GSM461348///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 150 (A)///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 317///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461348/GSM461348.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
      title: Ovarian Tumor, 321///geo_accession: GSM461380///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 45///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 321///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461380/GSM461380.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
      title: Ovarian Tumor, 324///geo_accession: GSM461373///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 59///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 324///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461373/GSM461373.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
       title: Ovarian Tumor, 332///geo_accession: GSM461349///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 7///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 332///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461349/GSM461349.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
      title: Ovarian Tumor, 345///geo_accession: GSM461392///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 18///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 345///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461392/GSM461392.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
 title: Ovarian Tumor, 349///geo_accession: GSM461350///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 144 (A)///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 349///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461350/GSM461350.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
 title: Ovarian Tumor, 351///geo_accession: GSM461351///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 142 (A)///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 351///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461351/GSM461351.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
      title: Ovarian Tumor, 358///geo_accession: GSM461393///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 21///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 358///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461393/GSM461393.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
      title: Ovarian Tumor, 367///geo_accession: GSM461381///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 16///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 367///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461381/GSM461381.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
      title: Ovarian Tumor, 377///geo_accession: GSM461374///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 12///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 377///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461374/GSM461374.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
      title: Ovarian Tumor, 380///geo_accession: GSM461375///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 57///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 380///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461375/GSM461375.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
      title: Ovarian Tumor, 386///geo_accession: GSM461352///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 95///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 386///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461352/GSM461352.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
 title: Ovarian Tumor, 388///geo_accession: GSM461353///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 132 (A)///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 388///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461353/GSM461353.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
      title: Ovarian Tumor, 389///geo_accession: GSM461354///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 13///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 389///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461354/GSM461354.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
      title: Ovarian Tumor, 394///geo_accession: GSM461382///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 16///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 394///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461382/GSM461382.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
      title: Ovarian Tumor, 396///geo_accession: GSM461376///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 21///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 396///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461376/GSM461376.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
       title: Ovarian Tumor, 402///geo_accession: GSM461355///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 8///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 402///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461355/GSM461355.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
 title: Ovarian Tumor, 410///geo_accession: GSM461356///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 150 (A)///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 410///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461356/GSM461356.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
     title: Ovarian Tumor, 412///geo_accession: GSM461357///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 113///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 412///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461357/GSM461357.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
      title: Ovarian Tumor, 434///geo_accession: GSM461358///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 72///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 434///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461358/GSM461358.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
     title: Ovarian Tumor, 443///geo_accession: GSM461377///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 111///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 443///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461377/GSM461377.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
      title: Ovarian Tumor, 461///geo_accession: GSM461394///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 32///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 461///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461394/GSM461394.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
      title: Ovarian Tumor, 467///geo_accession: GSM461359///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 11///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 467///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461359/GSM461359.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
      title: Ovarian Tumor, 477///geo_accession: GSM461383///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 21///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 477///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461383/GSM461383.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
      title: Ovarian Tumor, 486///geo_accession: GSM461395///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 21///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 486///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461395/GSM461395.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
  title: Ovarian Tumor, 629///geo_accession: GSM461360///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 50 (A)///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 629///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461360/GSM461360.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
      title: Ovarian Tumor, 631///geo_accession: GSM461396///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 30///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 631///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461396/GSM461396.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
      title: Ovarian Tumor, 656///geo_accession: GSM461384///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 25///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 656///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461384/GSM461384.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
      title: Ovarian Tumor, 662///geo_accession: GSM461370///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 23///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 662///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461370/GSM461370.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
      title: Ovarian Tumor, 692///geo_accession: GSM461397///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 35///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 692///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461397/GSM461397.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
      title: Ovarian Tumor, 694///geo_accession: GSM461385///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 33///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 694///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461385/GSM461385.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
      title: Ovarian Tumor, 702///geo_accession: GSM461361///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 13///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 702///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461361/GSM461361.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
       title: Ovarian Tumor, 714///geo_accession: GSM461362///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 8///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 714///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461362/GSM461362.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
      title: Ovarian Tumor, 715///geo_accession: GSM461386///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 33///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 715///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461386/GSM461386.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
      title: Ovarian Tumor, 718///geo_accession: GSM461398///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 26///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 718///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461398/GSM461398.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
      title: Ovarian Tumor, 744///geo_accession: GSM461378///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 14///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 744///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461378/GSM461378.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
      title: Ovarian Tumor, 765///geo_accession: GSM461363///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 12///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 765///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461363/GSM461363.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
      title: Ovarian Tumor, 778///geo_accession: GSM461399///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 16///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 778///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461399/GSM461399.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
      title: Ovarian Tumor, 780///geo_accession: GSM461364///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 11///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 780///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461364/GSM461364.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
  title: Ovarian Tumor, 786///geo_accession: GSM461387///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 48 (A)///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 786///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461387/GSM461387.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
      title: Ovarian Tumor, 794///geo_accession: GSM461388///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 15///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 794///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461388/GSM461388.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
       title: Ovarian Tumor, 799///geo_accession: GSM461365///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 5///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 799///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461365/GSM461365.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
      title: Ovarian Tumor, 800///geo_accession: GSM461371///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 36///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 800///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461371/GSM461371.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
       title: Ovarian Tumor, 872///geo_accession: GSM461366///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 9///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 872///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461366/GSM461366.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
  title: Ovarian Tumor, 934///geo_accession: GSM461372///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 36 (A)///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 934///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461372/GSM461372.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
      title: Ovarian Tumor, 970///geo_accession: GSM461389///status: Public on Oct 17 2009///submission_date: Oct 11 2009///last_update_date: Jan 19 2011///type: RNA///channel_count: 1///source_name_ch1: papillary serous ovarian adenocarcinoma///organism_ch1: Homo sapiens///characteristics_ch1: tissue: papillary serous ovarian adenocarcinoma///characteristics_ch1.1: tumor stage: late///characteristics_ch1.2: tumor grade: high///characteristics_ch1.3: surv data: 18///treatment_protocol_ch1: All specimens were subjected to laser-based microdissection and analyzed as pure, microdissected epithelial cell populations.///molecule_ch1: total RNA///extract_protocol_ch1: RNeasy Micro Kit according to the manufacturers protocol (Qiagen Inc., Valencia, CA).///label_ch1: biotin///label_protocol_ch1: Two rounds of amplification were completed according to standard Affymetrix Two-Cycle Amplification protocol using 25ng of total RNA.///taxid_ch1: 9606///hyb_protocol: A 15 microgram aliquot of amplified biotinylated RNA was hybridized to a Human U133 Plus 2.0 GeneChip array (Affymetrix, Santa Clara, CA).///scan_protocol: Arrays were scanned using the laser confocal GeneChip Scanner 3000 (Affymetrix).///description: Gene expression data from tumor.///description.1: 970///data_processing: Low-level analysis included array normalization and estimation of expression level using an invariant set of probe sets to adjust the overall signal level of the arrays to the same level. Next, a model-based PM-only approach established gene expression levels using dChip software.    A modified semi-supervised method was applied in two stages: (1) supervised dimension reduction by fitting a univariate Cox model that included a jackknifing step to each gene, so only those genes with a consistently large Cox score were included in the signature. Among 200, 100, or 300 probe sets, 200 yielded an optimally sized predictor; (2) In stage 2, the dimensions of the data set were reduced from 200 to 5 by PC analysis. The number of PCs was set at 5 capturing 90% of the expression data. 4 or 6 PCs were investigated to see whether this parameter affected the results. A prediction model was then built using multivariate Cox regression, where independent variables included the first 5 PCs and debulking status.///data_processing.1: Standard leave-one-out validation evaluated the prediction model reserving one sample for testing, and using the remaining 52 patients to establish the prediction model following the method described above. The reserved patient had no contribution to the prediction model yielding 53 separate predicted hazards. Patients were equally divided into low and high-risk groups according to whether their hazard was less or greater than the sample median and were compared by Kaplan Meier analysis with a non-parametric log-rank test.///platform_id: GPL570///contact_name: Michael,,Birrer///contact_email: mbirrer@partners.org///contact_phone: 6177244800///contact_laboratory: Surgical Oncology Research Labs///contact_department: Medicine///contact_institute: MGH///contact_address: 70 Blossom Street///contact_city: Boston///contact_state: MA///contact_zip.postal_code: 2114///contact_country: USA///supplementary_file: ftp://ftp.ncbi.nih.gov/pub/geo/DATA/supplementary/samples/GSM461nnn/GSM461389/GSM461389.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 

duplicates: 
                              GSE18520.GSE18520_GSM462649 
                                                        1 
GSE18520.GSE18520_GSM462649///GSE18520.GSE18520_GSM462650 
                                                        1 
                              GSE18520.GSE18520_GSM462650 
                                                        1 
                                                     NA's 
                                                       60 

}}
\keyword{datasets}
