\name{ GSE17260 }
\alias{ GSE17260 }
\docType{data}
\title{ Gene expression profile for predicting survival in advanced-stage serous ovarian cancer across two independent datasets. }
\description{ Advanced-stage ovarian cancer patients are generally treated with platinum/taxane-based chemotherapy after primary debulking surgery. However, there is a wide range of outcomes for individual patients. Therefore, the clinicopathological factors alone are insufficient for predicting prognosis. Our aim is to identify a progression-free survival (PFS)-related molecular profile for predicting survival of patients with advanced-stage serous ovarian cancer.Advanced-stage serous ovarian cancer tissues from 110 Japanese patients who underwent primary surgery and platinum/taxane-based chemotherapy were profiled using oligonucleotide microarrays. We selected 88 PFS-related genes by a univariate Cox model (p<0.01) and generated the prognostic index based on 88 PFS-related genes after adjustment of regression coefficients of the respective genes by ridge regression Cox model using 10-fold cross-validation. The prognostic index was independently associated with PFS time compared to other clinical factors in multivariate analysis [hazard ratio (HR), 3.72; 95\% confidence interval (CI), 2.66-5.43; p<0.0001]. In an external dataset, multivariate analysis revealed that this prognostic index was significantly correlated with PFS time (HR, 1.54; 95\% CI, 1.20-1.98; p = 0.0008). Furthermore, the correlation between the prognostic index and overall survival time was confirmed in the two independent external datasets (log rank test, p = 0.0010 and 0.0008).The prognostic ability of our index based on the 88-gene expression profile in ridge regression Cox hazard model was shown to be independent of other clinical factors in predicting cancer prognosis across two distinct datasets. Further study will be necessary to improve predictive accuracy of the prognostic index toward clinical application for evaluation of the risk of recurrence in patients with advanced-stage serous ovarian cancer. }
\usage{data( GSE17260_eset )}
\value{An expression set}
\format{
\preformatted{
experimentData(eset):
Experiment data
  Experimenter name: Yoshihara K, Tajima A, Yahata T, Kodama S, Fujiwara H, Suzuki M, Onishi Y, Hatae M, Sueyoshi K, Fujiwara H, Kudo Y, Kotera K, Masuzaki H, Tashiro H, Katabuchi H, Inoue I, Tanaka K.Gene expression profile for predicting survival in advanced-stage serous ovarian cancer across two independent datasets. PLoS One. 2010 Mar 12; 5(3):e9615. 
  Laboratory: Yoshihara, Tanaka 2010 
  Contact information:  
  Title: Gene expression profile for predicting survival in advanced-stage serous ovarian cancer across two independent datasets. 
  URL:  
  PMIDs: 20300634 

  Abstract: A 257 word abstract is available. Use 'abstract' method.
  Information is available on: preprocessing 
  notes:
   platform_title:     
      Agilent-012391 Whole Human Genome Oligo Microarray G4112A
   platform_shorttitle:     
      Agilent G4112A
   platform_summary:     
      hgug4112a
   platform_manufacturer:     
      Agilent
   platform_distribution:     
      commercial
   platform_accession:     
      GPL6848
   version:     
      2015-09-22 19:16:49

featureData(eset):
An object of class 'AnnotatedDataFrame'
  featureNames: A_23_P100001 A_23_P100011 ... A_32_P99902 (30936 total)
  varLabels: probeset gene EntrezGene.ID best_probe
  varMetadata: labelDescription

}}
\details{
\preformatted{
assayData: 30936 features, 110 samples
Platform type: 
Overall survival time-to-event summary (in years):
Call: survfit(formula = Surv(time, cens) ~ -1)

      n  events  median 0.95LCL 0.95UCL 
 110.00   46.00    4.44    4.03      NA 

--------------------------- 
Available sample meta-data: 
--------------------------- 

alt_sample_name: 
 Serous ovarian cancer 10 Serous ovarian cancer 100 Serous ovarian cancer 104 
                        1                         1                         1 
Serous ovarian cancer 106 Serous ovarian cancer 107 Serous ovarian cancer 108 
                        1                         1                         1 
Serous ovarian cancer 109  Serous ovarian cancer 11 Serous ovarian cancer 110 
                        1                         1                         1 
Serous ovarian cancer 111 Serous ovarian cancer 112 Serous ovarian cancer 113 
                        1                         1                         1 
Serous ovarian cancer 114 Serous ovarian cancer 115 Serous ovarian cancer 116 
                        1                         1                         1 
Serous ovarian cancer 117 Serous ovarian cancer 118 Serous ovarian cancer 119 
                        1                         1                         1 
 Serous ovarian cancer 12 Serous ovarian cancer 120 Serous ovarian cancer 122 
                        1                         1                         1 
Serous ovarian cancer 123 Serous ovarian cancer 127 Serous ovarian cancer 129 
                        1                         1                         1 
Serous ovarian cancer 130 Serous ovarian cancer 131 Serous ovarian cancer 132 
                        1                         1                         1 
Serous ovarian cancer 134 Serous ovarian cancer 136 Serous ovarian cancer 137 
                        1                         1                         1 
Serous ovarian cancer 139 Serous ovarian cancer 140 Serous ovarian cancer 143 
                        1                         1                         1 
Serous ovarian cancer 144 Serous ovarian cancer 145 Serous ovarian cancer 146 
                        1                         1                         1 
Serous ovarian cancer 148 Serous ovarian cancer 149  Serous ovarian cancer 15 
                        1                         1                         1 
Serous ovarian cancer 150 Serous ovarian cancer 151 Serous ovarian cancer 154 
                        1                         1                         1 
Serous ovarian cancer 156 Serous ovarian cancer 157  Serous ovarian cancer 16 
                        1                         1                         1 
Serous ovarian cancer 160  Serous ovarian cancer 17 Serous ovarian cancer 171 
                        1                         1                         1 
Serous ovarian cancer 172 Serous ovarian cancer 173 Serous ovarian cancer 174 
                        1                         1                         1 
Serous ovarian cancer 176 Serous ovarian cancer 178  Serous ovarian cancer 18 
                        1                         1                         1 
Serous ovarian cancer 182 Serous ovarian cancer 183 Serous ovarian cancer 184 
                        1                         1                         1 
Serous ovarian cancer 185 Serous ovarian cancer 186   Serous ovarian cancer 2 
                        1                         1                         1 
 Serous ovarian cancer 20  Serous ovarian cancer 22  Serous ovarian cancer 23 
                        1                         1                         1 
 Serous ovarian cancer 25  Serous ovarian cancer 27  Serous ovarian cancer 31 
                        1                         1                         1 
 Serous ovarian cancer 36  Serous ovarian cancer 37  Serous ovarian cancer 38 
                        1                         1                         1 
  Serous ovarian cancer 4  Serous ovarian cancer 41  Serous ovarian cancer 42 
                        1                         1                         1 
 Serous ovarian cancer 43  Serous ovarian cancer 44  Serous ovarian cancer 45 
                        1                         1                         1 
 Serous ovarian cancer 49  Serous ovarian cancer 50  Serous ovarian cancer 51 
                        1                         1                         1 
 Serous ovarian cancer 52  Serous ovarian cancer 53  Serous ovarian cancer 54 
                        1                         1                         1 
 Serous ovarian cancer 55  Serous ovarian cancer 56  Serous ovarian cancer 57 
                        1                         1                         1 
 Serous ovarian cancer 58   Serous ovarian cancer 6  Serous ovarian cancer 60 
                        1                         1                         1 
 Serous ovarian cancer 61  Serous ovarian cancer 62  Serous ovarian cancer 64 
                        1                         1                         1 
 Serous ovarian cancer 66  Serous ovarian cancer 67  Serous ovarian cancer 68 
                        1                         1                         1 
 Serous ovarian cancer 69   Serous ovarian cancer 7  Serous ovarian cancer 72 
                        1                         1                         1 
 Serous ovarian cancer 77  Serous ovarian cancer 79  Serous ovarian cancer 80 
                        1                         1                         1 
                  (Other) 
                       11 

sample_type: 
tumor 
  110 

histological_type: 
ser 
110 

primarysite: 
 ov 
110 

summarygrade: 
high  low 
  43   67 

summarystage: 
late 
 110 

tumorstage: 
 3  4 
93 17 

substage: 
   a    b    c NA's 
   6   18   69   17 

grade: 
 1  2  3 
26 41 43 

pltx: 
  y 
110 

tax: 
  y 
110 

days_to_tumor_recurrence: 
   Min. 1st Qu.  Median    Mean 3rd Qu.    Max. 
   30.0   285.0   510.0   673.9   870.0  2250.0 

recurrence_status: 
norecurrence   recurrence 
          34           76 

days_to_death: 
   Min. 1st Qu.  Median    Mean 3rd Qu.    Max. 
     30     660     915    1086    1530    2430 

vital_status: 
deceased   living 
      46       64 

debulking: 
   optimal suboptimal 
        57         53 

uncurated_author_metadata: 
                            title: Serous ovarian cancer 100///geo_accession: GSM432221///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 3///characteristics_ch1.1: Stage: IV///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 1///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 1///characteristics_ch1.6: death (1): 1///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432221/GSM432221.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                        title: Serous ovarian cancer 104///geo_accession: GSM432222///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 3///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 24///characteristics_ch1.4: recurrence (1): 0///characteristics_ch1.5: overall survival (m): 24///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432222/GSM432222.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
title: Serous ovarian cancer 106///geo_accession: GSM432223///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 15 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 3///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 15///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 26///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432223/GSM432223.txt.gz///data_row_count: 41000///relation: Reanalyzed by: GSM795125 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                        title: Serous ovarian cancer 107///geo_accession: GSM432224///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 3///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 17///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 33///characteristics_ch1.6: death (1): 1///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432224/GSM432224.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
    title: Serous ovarian cancer 108///geo_accession: GSM432225///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 15 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 1///characteristics_ch1.1: Stage: IIIb///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 37///characteristics_ch1.4: recurrence (1): 0///characteristics_ch1.5: overall survival (m): 37///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432225/GSM432225.txt.gz///data_row_count: 41000///relation: Reanalyzed by: GSM795126 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
 title: Serous ovarian cancer 109///geo_accession: GSM432226///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 15 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 3///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 7///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 20///characteristics_ch1.6: death (1): 1///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432226/GSM432226.txt.gz///data_row_count: 41000///relation: Reanalyzed by: GSM795127 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                             title: Serous ovarian cancer 10///geo_accession: GSM432220///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 3///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 55///characteristics_ch1.4: recurrence (1): 0///characteristics_ch1.5: overall survival (m): 55///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432220/GSM432220.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
title: Serous ovarian cancer 110///geo_accession: GSM432228///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 15 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 3///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 60///characteristics_ch1.4: recurrence (1): 0///characteristics_ch1.5: overall survival (m): 60///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432228/GSM432228.txt.gz///data_row_count: 41000///relation: Reanalyzed by: GSM795128 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
title: Serous ovarian cancer 111///geo_accession: GSM432229///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 15 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 2///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 18///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 57///characteristics_ch1.6: death (1): 1///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432229/GSM432229.txt.gz///data_row_count: 41000///relation: Reanalyzed by: GSM795129 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                            title: Serous ovarian cancer 112///geo_accession: GSM432230///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 2///characteristics_ch1.1: Stage: IIIa///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 48///characteristics_ch1.4: recurrence (1): 0///characteristics_ch1.5: overall survival (m): 48///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432230/GSM432230.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                        title: Serous ovarian cancer 113///geo_accession: GSM432231///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 1///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 43///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 69///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432231/GSM432231.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                         title: Serous ovarian cancer 114///geo_accession: GSM432232///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 1///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 8///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 27///characteristics_ch1.6: death (1): 1///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432232/GSM432232.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                        title: Serous ovarian cancer 115///geo_accession: GSM432233///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 3///characteristics_ch1.1: Stage: IIIb///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 12///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 69///characteristics_ch1.6: death (1): 1///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432233/GSM432233.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                        title: Serous ovarian cancer 116///geo_accession: GSM432234///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 1///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 14///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 51///characteristics_ch1.6: death (1): 1///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432234/GSM432234.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                            title: Serous ovarian cancer 117///geo_accession: GSM432235///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 2///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 17///characteristics_ch1.4: recurrence (1): 0///characteristics_ch1.5: overall survival (m): 17///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432235/GSM432235.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                         title: Serous ovarian cancer 118///geo_accession: GSM432236///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 2///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 8///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 17///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432236/GSM432236.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                            title: Serous ovarian cancer 119///geo_accession: GSM432237///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 3///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 17///characteristics_ch1.4: recurrence (1): 0///characteristics_ch1.5: overall survival (m): 17///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432237/GSM432237.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                             title: Serous ovarian cancer 11///geo_accession: GSM432227///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 2///characteristics_ch1.1: Stage: IIIb///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 46///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 47///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432227/GSM432227.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                           title: Serous ovarian cancer 120///geo_accession: GSM432239///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 2///characteristics_ch1.1: Stage: IV///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 6///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 52///characteristics_ch1.6: death (1): 1///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432239/GSM432239.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                         title: Serous ovarian cancer 122///geo_accession: GSM432240///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 2///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 8///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 25///characteristics_ch1.6: death (1): 1///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432240/GSM432240.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                        title: Serous ovarian cancer 123///geo_accession: GSM432242///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 2///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 15///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 22///characteristics_ch1.6: death (1): 1///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432242/GSM432242.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                        title: Serous ovarian cancer 127///geo_accession: GSM432243///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 3///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 23///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 53///characteristics_ch1.6: death (1): 1///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432243/GSM432243.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                            title: Serous ovarian cancer 129///geo_accession: GSM432244///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 3///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 12///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 13///characteristics_ch1.6: death (1): 1///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432244/GSM432244.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                              title: Serous ovarian cancer 12///geo_accession: GSM432238///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 3///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 8///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 24///characteristics_ch1.6: death (1): 1///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432238/GSM432238.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                        title: Serous ovarian cancer 130///geo_accession: GSM432245///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 3///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 25///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 79///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432245/GSM432245.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                            title: Serous ovarian cancer 131///geo_accession: GSM432246///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 1///characteristics_ch1.1: Stage: IIIa///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 74///characteristics_ch1.4: recurrence (1): 0///characteristics_ch1.5: overall survival (m): 74///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432246/GSM432246.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                            title: Serous ovarian cancer 132///geo_accession: GSM432247///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 3///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 75///characteristics_ch1.4: recurrence (1): 0///characteristics_ch1.5: overall survival (m): 75///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432247/GSM432247.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                            title: Serous ovarian cancer 134///geo_accession: GSM432248///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 2///characteristics_ch1.1: Stage: IIIa///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 30///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 62///characteristics_ch1.6: death (1): 1///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432248/GSM432248.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                            title: Serous ovarian cancer 136///geo_accession: GSM432249///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 2///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 64///characteristics_ch1.4: recurrence (1): 0///characteristics_ch1.5: overall survival (m): 64///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432249/GSM432249.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                            title: Serous ovarian cancer 137///geo_accession: GSM432250///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 3///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 46///characteristics_ch1.4: recurrence (1): 0///characteristics_ch1.5: overall survival (m): 46///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432250/GSM432250.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                            title: Serous ovarian cancer 139///geo_accession: GSM432251///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 3///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 35///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 44///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432251/GSM432251.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                          title: Serous ovarian cancer 140///geo_accession: GSM432252///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 3///characteristics_ch1.1: Stage: IV///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 14///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 28///characteristics_ch1.6: death (1): 1///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432252/GSM432252.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                         title: Serous ovarian cancer 143///geo_accession: GSM432253///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 2///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 6///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 33///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432253/GSM432253.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                          title: Serous ovarian cancer 144///geo_accession: GSM432254///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 3///characteristics_ch1.1: Stage: IV///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 26///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 31///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432254/GSM432254.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                        title: Serous ovarian cancer 145///geo_accession: GSM432255///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 3///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 24///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 28///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432255/GSM432255.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                        title: Serous ovarian cancer 146///geo_accession: GSM432256///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 3///characteristics_ch1.1: Stage: IIIb///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 24///characteristics_ch1.4: recurrence (1): 0///characteristics_ch1.5: overall survival (m): 24///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432256/GSM432256.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                            title: Serous ovarian cancer 148///geo_accession: GSM432257///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 3///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 15///characteristics_ch1.4: recurrence (1): 0///characteristics_ch1.5: overall survival (m): 15///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432257/GSM432257.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                            title: Serous ovarian cancer 149///geo_accession: GSM432258///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 2///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 15///characteristics_ch1.4: recurrence (1): 0///characteristics_ch1.5: overall survival (m): 15///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432258/GSM432258.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                        title: Serous ovarian cancer 150///geo_accession: GSM432260///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 2///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 14///characteristics_ch1.4: recurrence (1): 0///characteristics_ch1.5: overall survival (m): 14///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432260/GSM432260.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                            title: Serous ovarian cancer 151///geo_accession: GSM432261///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 1///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 13///characteristics_ch1.4: recurrence (1): 0///characteristics_ch1.5: overall survival (m): 13///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432261/GSM432261.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                              title: Serous ovarian cancer 154///geo_accession: GSM432262///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 2///characteristics_ch1.1: Stage: IIIb///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 9///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 9///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432262/GSM432262.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                            title: Serous ovarian cancer 156///geo_accession: GSM432263///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 3///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 29///characteristics_ch1.4: recurrence (1): 0///characteristics_ch1.5: overall survival (m): 29///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432263/GSM432263.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                            title: Serous ovarian cancer 157///geo_accession: GSM432264///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 2///characteristics_ch1.1: Stage: IIIb///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 26///characteristics_ch1.4: recurrence (1): 0///characteristics_ch1.5: overall survival (m): 26///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432264/GSM432264.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                             title: Serous ovarian cancer 15///geo_accession: GSM432259///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 2///characteristics_ch1.1: Stage: IIIb///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 18///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 49///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432259/GSM432259.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                        title: Serous ovarian cancer 160///geo_accession: GSM432266///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 2///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 16///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 20///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432266/GSM432266.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                             title: Serous ovarian cancer 16///geo_accession: GSM432265///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 1///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 13///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 49///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432265/GSM432265.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                         title: Serous ovarian cancer 171///geo_accession: GSM432268///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 3///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 3///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 15///characteristics_ch1.6: death (1): 1///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432268/GSM432268.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                         title: Serous ovarian cancer 172///geo_accession: GSM432269///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 3///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 2///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 12///characteristics_ch1.6: death (1): 1///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432269/GSM432269.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                         title: Serous ovarian cancer 173///geo_accession: GSM432270///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 2///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 5///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 26///characteristics_ch1.6: death (1): 1///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432270/GSM432270.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                              title: Serous ovarian cancer 174///geo_accession: GSM432271///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 3///characteristics_ch1.1: Stage: IV///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 23///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 41///characteristics_ch1.6: death (1): 1///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432271/GSM432271.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                           title: Serous ovarian cancer 176///geo_accession: GSM432272///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 3///characteristics_ch1.1: Stage: IV///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 1///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 11///characteristics_ch1.6: death (1): 1///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432272/GSM432272.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                              title: Serous ovarian cancer 178///geo_accession: GSM432273///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 3///characteristics_ch1.1: Stage: IV///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 17///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 47///characteristics_ch1.6: death (1): 1///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432273/GSM432273.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                             title: Serous ovarian cancer 17///geo_accession: GSM432267///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 2///characteristics_ch1.1: Stage: IIIb///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 47///characteristics_ch1.4: recurrence (1): 0///characteristics_ch1.5: overall survival (m): 47///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432267/GSM432267.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                          title: Serous ovarian cancer 182///geo_accession: GSM432275///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 2///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 1///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 1///characteristics_ch1.6: death (1): 1///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432275/GSM432275.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                        title: Serous ovarian cancer 183///geo_accession: GSM432276///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 2///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 72///characteristics_ch1.4: recurrence (1): 0///characteristics_ch1.5: overall survival (m): 72///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432276/GSM432276.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                            title: Serous ovarian cancer 184///geo_accession: GSM432277///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 2///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 65///characteristics_ch1.4: recurrence (1): 0///characteristics_ch1.5: overall survival (m): 65///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432277/GSM432277.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                            title: Serous ovarian cancer 185///geo_accession: GSM432278///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 3///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 20///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 33///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432278/GSM432278.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                            title: Serous ovarian cancer 186///geo_accession: GSM432279///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 3///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 12///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 29///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432279/GSM432279.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                             title: Serous ovarian cancer 18///geo_accession: GSM432274///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 1///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 50///characteristics_ch1.4: recurrence (1): 0///characteristics_ch1.5: overall survival (m): 50///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432274/GSM432274.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                             title: Serous ovarian cancer 20///geo_accession: GSM432281///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 2///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 70///characteristics_ch1.4: recurrence (1): 0///characteristics_ch1.5: overall survival (m): 70///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432281/GSM432281.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                         title: Serous ovarian cancer 22///geo_accession: GSM432282///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 2///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 15///characteristics_ch1.4: recurrence (1): 0///characteristics_ch1.5: overall survival (m): 15///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432282/GSM432282.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                           title: Serous ovarian cancer 23///geo_accession: GSM432283///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 2///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 2///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 8///characteristics_ch1.6: death (1): 1///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432283/GSM432283.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                             title: Serous ovarian cancer 25///geo_accession: GSM432284///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 1///characteristics_ch1.1: Stage: IIIb///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 29///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 80///characteristics_ch1.6: death (1): 1///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432284/GSM432284.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                          title: Serous ovarian cancer 27///geo_accession: GSM432285///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 1///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 8///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 11///characteristics_ch1.6: death (1): 1///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432285/GSM432285.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                          title: Serous ovarian cancer 2///geo_accession: GSM432280///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 2///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 18///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 49///characteristics_ch1.6: death (1): 1///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432280/GSM432280.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                               title: Serous ovarian cancer 31///geo_accession: GSM432286///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 3///characteristics_ch1.1: Stage: IIIb///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 2///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 5///characteristics_ch1.6: death (1): 1///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432286/GSM432286.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                               title: Serous ovarian cancer 36///geo_accession: GSM432287///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 1///characteristics_ch1.1: Stage: IV///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 23///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 28///characteristics_ch1.6: death (1): 1///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432287/GSM432287.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                             title: Serous ovarian cancer 37///geo_accession: GSM432288///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 2///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 21///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 25///characteristics_ch1.6: death (1): 1///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432288/GSM432288.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                             title: Serous ovarian cancer 38///geo_accession: GSM432289///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 2///characteristics_ch1.1: Stage: IIIb///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 15///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 64///characteristics_ch1.6: death (1): 1///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432289/GSM432289.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                             title: Serous ovarian cancer 41///geo_accession: GSM432291///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 1///characteristics_ch1.1: Stage: IIIb///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 61///characteristics_ch1.4: recurrence (1): 0///characteristics_ch1.5: overall survival (m): 61///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432291/GSM432291.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                         title: Serous ovarian cancer 42///geo_accession: GSM432292///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 3///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 17///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 46///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432292/GSM432292.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                          title: Serous ovarian cancer 43///geo_accession: GSM432293///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 2///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 7///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 34///characteristics_ch1.6: death (1): 1///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432293/GSM432293.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                          title: Serous ovarian cancer 44///geo_accession: GSM432294///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 3///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 4///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 17///characteristics_ch1.6: death (1): 1///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432294/GSM432294.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                             title: Serous ovarian cancer 45///geo_accession: GSM432295///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 1///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 35///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 39///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432295/GSM432295.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                         title: Serous ovarian cancer 49///geo_accession: GSM432296///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 2///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 33///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 37///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432296/GSM432296.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                           title: Serous ovarian cancer 4///geo_accession: GSM432290///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 2///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 2///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 41///characteristics_ch1.6: death (1): 1///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432290/GSM432290.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                              title: Serous ovarian cancer 50///geo_accession: GSM432297///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 3///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 8///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 33///characteristics_ch1.6: death (1): 1///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432297/GSM432297.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                         title: Serous ovarian cancer 51///geo_accession: GSM432298///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 2///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 14///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 25///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432298/GSM432298.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                            title: Serous ovarian cancer 52///geo_accession: GSM432299///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 3///characteristics_ch1.1: Stage: IV///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 4///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 24///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432299/GSM432299.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                            title: Serous ovarian cancer 53///geo_accession: GSM432300///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 3///characteristics_ch1.1: Stage: IV///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 9///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 23///characteristics_ch1.6: death (1): 1///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432300/GSM432300.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                             title: Serous ovarian cancer 54///geo_accession: GSM432301///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 1///characteristics_ch1.1: Stage: IIIb///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 30///characteristics_ch1.4: recurrence (1): 0///characteristics_ch1.5: overall survival (m): 30///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432301/GSM432301.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                             title: Serous ovarian cancer 55///geo_accession: GSM432302///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 3///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 30///characteristics_ch1.4: recurrence (1): 0///characteristics_ch1.5: overall survival (m): 30///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432302/GSM432302.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                         title: Serous ovarian cancer 56///geo_accession: GSM432303///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 1///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 14///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 29///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432303/GSM432303.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                             title: Serous ovarian cancer 57///geo_accession: GSM432304///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 3///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 12///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 23///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432304/GSM432304.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                             title: Serous ovarian cancer 58///geo_accession: GSM432305///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 1///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 13///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 20///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432305/GSM432305.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                           title: Serous ovarian cancer 60///geo_accession: GSM432307///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 2///characteristics_ch1.1: Stage: IV///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 47///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 81///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432307/GSM432307.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                         title: Serous ovarian cancer 61///geo_accession: GSM432308///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 1///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 26///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 74///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432308/GSM432308.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                         title: Serous ovarian cancer 62///geo_accession: GSM432309///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 2///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 11///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 26///characteristics_ch1.6: death (1): 1///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432309/GSM432309.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                             title: Serous ovarian cancer 64///geo_accession: GSM432310///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 2///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 26///characteristics_ch1.4: recurrence (1): 0///characteristics_ch1.5: overall survival (m): 26///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432310/GSM432310.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                             title: Serous ovarian cancer 66///geo_accession: GSM432311///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 1///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 19///characteristics_ch1.4: recurrence (1): 0///characteristics_ch1.5: overall survival (m): 19///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432311/GSM432311.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                             title: Serous ovarian cancer 67///geo_accession: GSM432312///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 1///characteristics_ch1.1: Stage: IIIb///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 22///characteristics_ch1.4: recurrence (1): 0///characteristics_ch1.5: overall survival (m): 22///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432312/GSM432312.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                             title: Serous ovarian cancer 68///geo_accession: GSM432313///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 1///characteristics_ch1.1: Stage: IIIa///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 22///characteristics_ch1.4: recurrence (1): 0///characteristics_ch1.5: overall survival (m): 22///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432313/GSM432313.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                             title: Serous ovarian cancer 69///geo_accession: GSM432314///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 1///characteristics_ch1.1: Stage: IIIb///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 21///characteristics_ch1.4: recurrence (1): 0///characteristics_ch1.5: overall survival (m): 21///characteristics_ch1.6: death (1): 0///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432314/GSM432314.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                              title: Serous ovarian cancer 6///geo_accession: GSM432306///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 3///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 12///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 32///characteristics_ch1.6: death (1): 1///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432306/GSM432306.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                             title: Serous ovarian cancer 72///geo_accession: GSM432316///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 2///characteristics_ch1.1: Stage: IIIb///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 13///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 26///characteristics_ch1.6: death (1): 1///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432316/GSM432316.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                           title: Serous ovarian cancer 77///geo_accession: GSM432317///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 3///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 1///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 1///characteristics_ch1.6: death (1): 1///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432317/GSM432317.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                             title: Serous ovarian cancer 79///geo_accession: GSM432318///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 1///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 16///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 27///characteristics_ch1.6: death (1): 1///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432318/GSM432318.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                              title: Serous ovarian cancer 7///geo_accession: GSM432315///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 1///characteristics_ch1.1: Stage: IIIa///characteristics_ch1.2: cytoreductive surgery: optimal///characteristics_ch1.3: progression-free survival (m): 19///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 42///characteristics_ch1.6: death (1): 1///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432315/GSM432315.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                         title: Serous ovarian cancer 80///geo_accession: GSM432319///status: Public on Mar 22 2010///submission_date: Jul 22 2009///last_update_date: Sep 13 2011///type: RNA///channel_count: 1///source_name_ch1: serous ovarian cancer///organism_ch1: Homo sapiens///characteristics_ch1: tumor grade: 2///characteristics_ch1.1: Stage: IIIc///characteristics_ch1.2: cytoreductive surgery: not optimal///characteristics_ch1.3: progression-free survival (m): 12///characteristics_ch1.4: recurrence (1): 1///characteristics_ch1.5: overall survival (m): 68///characteristics_ch1.6: death (1): 1///characteristics_ch1.7: disease state: serous ovarian cancer///characteristics_ch1.8: tissue: cancer tumor///molecule_ch1: total RNA///extract_protocol_ch1: Total RNA was extracted from tissue samples using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) following the manufacturer's recommendations. RNA was examined with a 2100 Bioanalyzer (Agilent Technologies Inc., Palo Alto, CA, USA) using an RNA 6000 Nano LabChip (Agilent Technologies)///label_ch1: Cy3///label_protocol_ch1: Cyanine-3 (Cy3) labeled cRNA was prepared from 0.5 ug RNA using the Quick Amp Labering Kit, one-color (Agilent Technologies) according to the manufacturer's instructions, followed by RNAeasy column purification (QIAGEN, Valencia, CA).///taxid_ch1: 9606///hyb_protocol: Cy3-labeled cRNAs (1.65 g) were hybridized for 17 hours at 65C to an Agilent Whole Human Genome Oligo Microarray in a rotating Agilent hybridization oven.///scan_protocol: The hybridized microarray was washed and then scanned in Cy3 channel with the Agilent DNA Microarray Scanner (model G2565AA).///description: n/a///data_processing: Signal intensity per spot was generated from the scanned image with Feature Extraction Software ver8.5 (Agilent Technologies) in the default settings. Data normalization was performed in GeneSpring GX 10 (Agilent Technologies) as follows: (i) Threshold raw signals were set to 1.0. (ii) Quantile normalization was chosen as normalized algorithm. (iii) Baseline was transformed to median of all samples.///platform_id: GPL6480///contact_name: Kosuke,,Yoshihara///contact_email: yoshikou@med.niigata-u.ac.jp///contact_department: Obstetrics and Gynecology///contact_institute: Niigata University///contact_address: 1-757 Asahimachi-dori///contact_city: Niigata///contact_zip.postal_code: 951-8510///contact_country: Japan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM432nnn/GSM432319/GSM432319.txt.gz///data_row_count: 41000///relation:  
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   (Other) 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     11 

}}
\keyword{datasets}
