\name{ GSE44104_eset }
\alias{ GSE44104_eset }
\docType{data}
\title{ COL11A1 promotes tumor progression and predicts poor clinical outcome in ovarian cancer. }
\description{ Biomarkers that predict disease progression might assist the development of better therapeutic strategies for aggressive cancers, such as ovarian cancer. Here, we investigated the role of collagen type XI alpha 1 (COL11A1) in cell invasiveness and tumor formation and the prognostic impact of COL11A1 expression in ovarian cancer. Microarray analysis suggested that COL11A1 is a disease progression-associated gene that is linked to ovarian cancer recurrence and poor survival. Small interference RNA-mediated specific reduction in COL11A1 protein levels suppressed the invasive ability and oncogenic potential of ovarian cancer cells and decreased tumor formation and lung colonization in mouse xenografts. A combination of experimental approaches, including real-time RT-PCR, casein zymography and chromatin immunoprecipitation (ChIP) assays, showed that COL11A1 knockdown attenuated MMP3 expression and suppressed binding of Ets-1 to its putative MMP3 promoter-binding site, suggesting that the Ets-1-MMP3 axis is upregulated by COL11A1. Transforming growth factor (TGF)-beta (TGF-??1) treatment triggers the activation of smad2 signaling cascades, leading to activation of COL11A1 and MMP3. Pharmacological inhibition of MMP3 abrogated the TGF-??1-triggered, COL11A1-dependent cell invasiveness. Furthermore, the NF-YA-binding site on the COL11A1 promoter was identified as the major determinant of TGF-??1-dependent COL11A1 activation. Analysis of 88 ovarian cancer patients indicated that high COL11A1 mRNA levels are associated with advanced disease stage. The 5-year recurrence-free and overall survival rates were significantly lower (P=0.006 and P=0.018, respectively) among patients with high expression levels of tissue COL11A1 mRNA compared with those with low expression. We conclude that COL11A1 may promote tumor aggressiveness via the TGF-??1-MMP3 axis and that COL11A1 expression can predict clinical outcome in ovarian cancer patients. }
\usage{data( GSE44104_eset )}
\format{
\preformatted{
experimentData(eset):
Experiment data
  Experimenter name: Wu Y, Chang T, Huang Y, Huang H, Chou C 
  Laboratory: Wu, Chou 2013 
  Contact information:  
  Title: COL11A1 promotes tumor progression and predicts poor clinical outcome in ovarian cancer. 
  URL:  
  PMIDs: 23934190 

  Abstract: A 260 word abstract is available. Use 'abstract' method.
  Information is available on: preprocessing 
  notes:
   platform_title:     
      [HG-U133_Plus_2] Affymetrix Human Genome U133 Plus 2.0 Array
   platform_shorttitle:     
      Affymetrix HG-U133Plus2
   platform_summary:     
      hgu133plus2
   platform_manufacturer:     
      Affymetrix
   platform_distribution:     
      commercial
   platform_accession:     
      GPL570
   platform_technology:     
      in situ oligonucleotide
   version:     
      2015-09-22 20:02:05

featureData(eset):
An object of class 'AnnotatedDataFrame'
  featureNames: 1007_s_at 1053_at ... AFFX-HUMISGF3A/M97935_MB_at
    (42447 total)
  varLabels: probeset gene EntrezGene.ID best_probe
  varMetadata: labelDescription

}}
\details{
\preformatted{
assayData: 42447 features, 60 samples
Platform type: 
--------------------------- 
Available sample meta-data: 
--------------------------- 

alt_sample_name: 
Tc_113  Tc_48  Tc_49  Tc_51  Tc_56  Tc_59  Tc_61  Tc_63  Tc_64  Tc_65  Tc_74 
     1      1      1      1      1      1      1      1      1      1      1 
 Tc_94  Te_69  Te_77  Te_78  Te_79  Te_84  Te_87  Te_89  Te_90  Te_91  Te_92 
     1      1      1      1      1      1      1      1      1      1      1 
 Te_93 Tm_101 Tm_102 Tm_106 Tm_107 Tm_110  Tm_95  Tm_96  Tm_97  Tm_98  Ts_11 
     1      1      1      1      1      1      1      1      1      1      1 
 Ts_14  Ts_15  Ts_17  Ts_19   Ts_2  Ts_20  Ts_21  Ts_23  Ts_24  Ts_26  Ts_28 
     1      1      1      1      1      1      1      1      1      1      1 
  Ts_3  Ts_31  Ts_32  Ts_34  Ts_35  Ts_36  Ts_37  Ts_39   Ts_4  Ts_41  Ts_43 
     1      1      1      1      1      1      1      1      1      1      1 
 Ts_45  Ts_46  Ts_47   Ts_5   Ts_8 
     1      1      1      1      1 

sample_type: 
tumor 
   60 

histological_type: 
clearcell      endo  mucinous       ser 
       12        11         9        28 

summarystage: 
early  late 
   25    35 

tumorstage: 
 1  2  3  4 
17  8 30  5 

recurrence_status: 
norecurrence   recurrence 
          40           20 

os_binary: 
 long short 
   44    16 

relapse_binary: 
 long short 
   40    20 

batch: 
2010-09-07 2010-09-08 2010-10-14 2010-12-10 2010-12-14 
        20          2         18         16          4 

uncurated_author_metadata: 
title: Tc_113///geo_accession: GSM1078972///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: clear cell///characteristics_ch1.2: advanced disease stage: stage III///characteristics_ch1.3: recurrence status: non-recurrence///characteristics_ch1.4: survival/death: survival///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: c113///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1078nnn/GSM1078972/GSM1078972_c113.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
          title: Tc_48///geo_accession: GSM1078973///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: clear cell///characteristics_ch1.2: advanced disease stage: stage III///characteristics_ch1.3: recurrence status: recurrence///characteristics_ch1.4: survival/death: death///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: c48///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1078nnn/GSM1078973/GSM1078973_c48.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
     title: Tc_49///geo_accession: GSM1078974///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: clear cell///characteristics_ch1.2: advanced disease stage: stage I///characteristics_ch1.3: recurrence status: non-recurrence///characteristics_ch1.4: survival/death: survival///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: c49///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1078nnn/GSM1078974/GSM1078974_c49.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
   title: Tc_51///geo_accession: GSM1078975///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: clear cell///characteristics_ch1.2: advanced disease stage: stage III///characteristics_ch1.3: recurrence status: non-recurrence///characteristics_ch1.4: survival/death: survival///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: c51///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1078nnn/GSM1078975/GSM1078975_c51.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
     title: Tc_56///geo_accession: GSM1078976///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: clear cell///characteristics_ch1.2: advanced disease stage: stage I///characteristics_ch1.3: recurrence status: non-recurrence///characteristics_ch1.4: survival/death: survival///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: c56///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1078nnn/GSM1078976/GSM1078976_c56.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
     title: Tc_59///geo_accession: GSM1078977///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: clear cell///characteristics_ch1.2: advanced disease stage: stage I///characteristics_ch1.3: recurrence status: non-recurrence///characteristics_ch1.4: survival/death: survival///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: c59///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1078nnn/GSM1078977/GSM1078977_c59.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
   title: Tc_61///geo_accession: GSM1078978///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: clear cell///characteristics_ch1.2: advanced disease stage: stage III///characteristics_ch1.3: recurrence status: non-recurrence///characteristics_ch1.4: survival/death: survival///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: c61///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1078nnn/GSM1078978/GSM1078978_c61.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
     title: Tc_63///geo_accession: GSM1078979///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: clear cell///characteristics_ch1.2: advanced disease stage: stage I///characteristics_ch1.3: recurrence status: non-recurrence///characteristics_ch1.4: survival/death: survival///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: c63///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1078nnn/GSM1078979/GSM1078979_c63.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
     title: Tc_64///geo_accession: GSM1078980///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: clear cell///characteristics_ch1.2: advanced disease stage: stage I///characteristics_ch1.3: recurrence status: non-recurrence///characteristics_ch1.4: survival/death: survival///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: c64///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1078nnn/GSM1078980/GSM1078980_c64.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
        title: Tc_65///geo_accession: GSM1078981///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: clear cell///characteristics_ch1.2: advanced disease stage: stage II///characteristics_ch1.3: recurrence status: recurrence///characteristics_ch1.4: survival/death: survival///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: c65///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1078nnn/GSM1078981/GSM1078981_c65.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
        title: Tc_74///geo_accession: GSM1078982///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: clear cell///characteristics_ch1.2: advanced disease stage: stage IV///characteristics_ch1.3: recurrence status: recurrence///characteristics_ch1.4: survival/death: survival///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: c74///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1078nnn/GSM1078982/GSM1078982_c74.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
       title: Tc_94///geo_accession: GSM1078983///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: clear cell///characteristics_ch1.2: advanced disease stage: stage IV///characteristics_ch1.3: recurrence status: non-recurrence///characteristics_ch1.4: survival/death: death///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: c94///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1078nnn/GSM1078983/GSM1078983_c94.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
      title: Te_69///geo_accession: GSM1078984///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: endometrioid///characteristics_ch1.2: advanced disease stage: stage I///characteristics_ch1.3: recurrence status: non-recurrence///characteristics_ch1.4: survival/death: death///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: e69///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1078nnn/GSM1078984/GSM1078984_e69.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
     title: Te_77///geo_accession: GSM1078985///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: endometrioid///characteristics_ch1.2: advanced disease stage: stage II///characteristics_ch1.3: recurrence status: non-recurrence///characteristics_ch1.4: survival/death: death///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: e77///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1078nnn/GSM1078985/GSM1078985_e77.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
   title: Te_78///geo_accession: GSM1078986///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: endometrioid///characteristics_ch1.2: advanced disease stage: stage I///characteristics_ch1.3: recurrence status: non-recurrence///characteristics_ch1.4: survival/death: survival///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: e78///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1078nnn/GSM1078986/GSM1078986_e78.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
   title: Te_79///geo_accession: GSM1078987///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: endometrioid///characteristics_ch1.2: advanced disease stage: stage I///characteristics_ch1.3: recurrence status: non-recurrence///characteristics_ch1.4: survival/death: survival///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: e79///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1078nnn/GSM1078987/GSM1078987_e79.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
  title: Te_84///geo_accession: GSM1078988///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: endometrioid///characteristics_ch1.2: advanced disease stage: stage II///characteristics_ch1.3: recurrence status: non-recurrence///characteristics_ch1.4: survival/death: survival///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: e84///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1078nnn/GSM1078988/GSM1078988_e84.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
 title: Te_87///geo_accession: GSM1078989///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: endometrioid///characteristics_ch1.2: advanced disease stage: stage III///characteristics_ch1.3: recurrence status: non-recurrence///characteristics_ch1.4: survival/death: survival///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: e87///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1078nnn/GSM1078989/GSM1078989_e87.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
   title: Te_89///geo_accession: GSM1078990///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: endometrioid///characteristics_ch1.2: advanced disease stage: stage I///characteristics_ch1.3: recurrence status: non-recurrence///characteristics_ch1.4: survival/death: survival///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: e89///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1078nnn/GSM1078990/GSM1078990_e89.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
  title: Te_90///geo_accession: GSM1078991///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: endometrioid///characteristics_ch1.2: advanced disease stage: stage IV///characteristics_ch1.3: recurrence status: non-recurrence///characteristics_ch1.4: survival/death: survival///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: e90///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1078nnn/GSM1078991/GSM1078991_e90.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
        title: Te_91///geo_accession: GSM1078992///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: endometrioid///characteristics_ch1.2: advanced disease stage: stage III///characteristics_ch1.3: recurrence status: recurrence///characteristics_ch1.4: survival/death: death///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: e91///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1078nnn/GSM1078992/GSM1078992_e91.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
   title: Te_92///geo_accession: GSM1078993///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: endometrioid///characteristics_ch1.2: advanced disease stage: stage I///characteristics_ch1.3: recurrence status: non-recurrence///characteristics_ch1.4: survival/death: survival///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: e92///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1078nnn/GSM1078993/GSM1078993_e92.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
  title: Te_93///geo_accession: GSM1078994///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: endometrioid///characteristics_ch1.2: advanced disease stage: stage II///characteristics_ch1.3: recurrence status: non-recurrence///characteristics_ch1.4: survival/death: survival///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: e93///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1078nnn/GSM1078994/GSM1078994_e93.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
     title: Tm_101///geo_accession: GSM1078995///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: mucinous///characteristics_ch1.2: advanced disease stage: stage III///characteristics_ch1.3: recurrence status: non-recurrence///characteristics_ch1.4: survival/death: death///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: m101///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1078nnn/GSM1078995/GSM1078995_m101.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
    title: Tm_102///geo_accession: GSM1078996///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: mucinous///characteristics_ch1.2: advanced disease stage: stage I///characteristics_ch1.3: recurrence status: non-recurrence///characteristics_ch1.4: survival/death: survival///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: m102///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1078nnn/GSM1078996/GSM1078996_m102.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
    title: Tm_106///geo_accession: GSM1078997///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: mucinous///characteristics_ch1.2: advanced disease stage: stage I///characteristics_ch1.3: recurrence status: non-recurrence///characteristics_ch1.4: survival/death: survival///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: m106///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1078nnn/GSM1078997/GSM1078997_m106.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
  title: Tm_107///geo_accession: GSM1078998///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: mucinous///characteristics_ch1.2: advanced disease stage: stage III///characteristics_ch1.3: recurrence status: non-recurrence///characteristics_ch1.4: survival/death: survival///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: m107///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1078nnn/GSM1078998/GSM1078998_m107.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
    title: Tm_110///geo_accession: GSM1078999///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: mucinous///characteristics_ch1.2: advanced disease stage: stage I///characteristics_ch1.3: recurrence status: non-recurrence///characteristics_ch1.4: survival/death: survival///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: m110///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1078nnn/GSM1078999/GSM1078999_m110.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
       title: Tm_95///geo_accession: GSM1079000///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: mucinous///characteristics_ch1.2: advanced disease stage: stage I///characteristics_ch1.3: recurrence status: non-recurrence///characteristics_ch1.4: survival/death: survival///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: m95///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1079nnn/GSM1079000/GSM1079000_m95.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
      title: Tm_96///geo_accession: GSM1079001///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: mucinous///characteristics_ch1.2: advanced disease stage: stage II///characteristics_ch1.3: recurrence status: non-recurrence///characteristics_ch1.4: survival/death: survival///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: m96///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1079nnn/GSM1079001/GSM1079001_m96.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
       title: Tm_97///geo_accession: GSM1079002///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: mucinous///characteristics_ch1.2: advanced disease stage: stage I///characteristics_ch1.3: recurrence status: non-recurrence///characteristics_ch1.4: survival/death: survival///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: m97///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1079nnn/GSM1079002/GSM1079002_m97.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
             title: Tm_98///geo_accession: GSM1079003///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: mucinous///characteristics_ch1.2: advanced disease stage: stage II///characteristics_ch1.3: recurrence status: recurrence///characteristics_ch1.4: survival/death: death///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: m98///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1079nnn/GSM1079003/GSM1079003_m98.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
           title: Ts_11///geo_accession: GSM1079004///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: serous///characteristics_ch1.2: advanced disease stage: stage III///characteristics_ch1.3: recurrence status: recurrence///characteristics_ch1.4: survival/death: survival///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: s11///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1079nnn/GSM1079004/GSM1079004_s11.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
              title: Ts_14///geo_accession: GSM1079005///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: serous///characteristics_ch1.2: advanced disease stage: stage III///characteristics_ch1.3: recurrence status: recurrence///characteristics_ch1.4: survival/death: death///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: s14///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1079nnn/GSM1079005/GSM1079005_s14.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
         title: Ts_15///geo_accession: GSM1079006///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: serous///characteristics_ch1.2: advanced disease stage: stage I///characteristics_ch1.3: recurrence status: non-recurrence///characteristics_ch1.4: survival/death: survival///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: s15///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1079nnn/GSM1079006/GSM1079006_s15.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                title: Ts_17///geo_accession: GSM1079007///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: serous///characteristics_ch1.2: advanced disease stage: stage I///characteristics_ch1.3: recurrence status: recurrence///characteristics_ch1.4: survival/death: death///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: s17///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1079nnn/GSM1079007/GSM1079007_s17.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
              title: Ts_19///geo_accession: GSM1079008///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: serous///characteristics_ch1.2: advanced disease stage: stage III///characteristics_ch1.3: recurrence status: recurrence///characteristics_ch1.4: survival/death: death///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: s19///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1079nnn/GSM1079008/GSM1079008_s19.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
          title: Ts_20///geo_accession: GSM1079009///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: serous///characteristics_ch1.2: advanced disease stage: stage III///characteristics_ch1.3: recurrence status: non-recurrence///characteristics_ch1.4: survival/death: death///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: s20///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1079nnn/GSM1079009/GSM1079009_s20.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
           title: Ts_21///geo_accession: GSM1079010///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: serous///characteristics_ch1.2: advanced disease stage: stage III///characteristics_ch1.3: recurrence status: recurrence///characteristics_ch1.4: survival/death: survival///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: s21///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1079nnn/GSM1079010/GSM1079010_s21.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
              title: Ts_23///geo_accession: GSM1079011///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: serous///characteristics_ch1.2: advanced disease stage: stage III///characteristics_ch1.3: recurrence status: recurrence///characteristics_ch1.4: survival/death: death///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: s23///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1079nnn/GSM1079011/GSM1079011_s23.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
           title: Ts_24///geo_accession: GSM1079012///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: serous///characteristics_ch1.2: advanced disease stage: stage III///characteristics_ch1.3: recurrence status: recurrence///characteristics_ch1.4: survival/death: survival///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: s24///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1079nnn/GSM1079012/GSM1079012_s24.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
              title: Ts_26///geo_accession: GSM1079013///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: serous///characteristics_ch1.2: advanced disease stage: stage III///characteristics_ch1.3: recurrence status: recurrence///characteristics_ch1.4: survival/death: death///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: s26///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1079nnn/GSM1079013/GSM1079013_s26.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
       title: Ts_28///geo_accession: GSM1079014///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: serous///characteristics_ch1.2: advanced disease stage: stage III///characteristics_ch1.3: recurrence status: non-recurrence///characteristics_ch1.4: survival/death: survival///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: s28///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1079nnn/GSM1079014/GSM1079014_s28.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                 title: Ts_2///geo_accession: GSM1079015///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: serous///characteristics_ch1.2: advanced disease stage: stage III///characteristics_ch1.3: recurrence status: recurrence///characteristics_ch1.4: survival/death: death///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: s2///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1079nnn/GSM1079015/GSM1079015_s2.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
       title: Ts_31///geo_accession: GSM1079016///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: serous///characteristics_ch1.2: advanced disease stage: stage III///characteristics_ch1.3: recurrence status: non-recurrence///characteristics_ch1.4: survival/death: survival///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: s31///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1079nnn/GSM1079016/GSM1079016_s31.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
       title: Ts_32///geo_accession: GSM1079017///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: serous///characteristics_ch1.2: advanced disease stage: stage III///characteristics_ch1.3: recurrence status: non-recurrence///characteristics_ch1.4: survival/death: survival///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: s32///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1079nnn/GSM1079017/GSM1079017_s32.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
       title: Ts_34///geo_accession: GSM1079018///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: serous///characteristics_ch1.2: advanced disease stage: stage III///characteristics_ch1.3: recurrence status: non-recurrence///characteristics_ch1.4: survival/death: survival///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: s34///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1079nnn/GSM1079018/GSM1079018_s34.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
       title: Ts_35///geo_accession: GSM1079019///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: serous///characteristics_ch1.2: advanced disease stage: stage III///characteristics_ch1.3: recurrence status: non-recurrence///characteristics_ch1.4: survival/death: survival///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: s35///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1079nnn/GSM1079019/GSM1079019_s35.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
           title: Ts_36///geo_accession: GSM1079020///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: serous///characteristics_ch1.2: advanced disease stage: stage III///characteristics_ch1.3: recurrence status: recurrence///characteristics_ch1.4: survival/death: survival///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: s36///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1079nnn/GSM1079020/GSM1079020_s36.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
       title: Ts_37///geo_accession: GSM1079021///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: serous///characteristics_ch1.2: advanced disease stage: stage III///characteristics_ch1.3: recurrence status: non-recurrence///characteristics_ch1.4: survival/death: survival///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: s37///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1079nnn/GSM1079021/GSM1079021_s37.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
           title: Ts_39///geo_accession: GSM1079022///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: serous///characteristics_ch1.2: advanced disease stage: stage III///characteristics_ch1.3: recurrence status: recurrence///characteristics_ch1.4: survival/death: survival///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: s39///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1079nnn/GSM1079022/GSM1079022_s39.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
              title: Ts_3///geo_accession: GSM1079023///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: serous///characteristics_ch1.2: advanced disease stage: stage III///characteristics_ch1.3: recurrence status: recurrence///characteristics_ch1.4: survival/death: survival///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: s3///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1079nnn/GSM1079023/GSM1079023_s3.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
       title: Ts_41///geo_accession: GSM1079024///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: serous///characteristics_ch1.2: advanced disease stage: stage III///characteristics_ch1.3: recurrence status: non-recurrence///characteristics_ch1.4: survival/death: survival///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: s41///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1079nnn/GSM1079024/GSM1079024_s41.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
        title: Ts_43///geo_accession: GSM1079025///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: serous///characteristics_ch1.2: advanced disease stage: stage II///characteristics_ch1.3: recurrence status: non-recurrence///characteristics_ch1.4: survival/death: survival///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: s43///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1079nnn/GSM1079025/GSM1079025_s43.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
        title: Ts_45///geo_accession: GSM1079026///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: serous///characteristics_ch1.2: advanced disease stage: stage IV///characteristics_ch1.3: recurrence status: non-recurrence///characteristics_ch1.4: survival/death: survival///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: s45///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1079nnn/GSM1079026/GSM1079026_s45.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
        title: Ts_46///geo_accession: GSM1079027///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: serous///characteristics_ch1.2: advanced disease stage: stage II///characteristics_ch1.3: recurrence status: non-recurrence///characteristics_ch1.4: survival/death: survival///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: s46///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1079nnn/GSM1079027/GSM1079027_s46.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
       title: Ts_47///geo_accession: GSM1079028///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: serous///characteristics_ch1.2: advanced disease stage: stage III///characteristics_ch1.3: recurrence status: non-recurrence///characteristics_ch1.4: survival/death: survival///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: s47///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1079nnn/GSM1079028/GSM1079028_s47.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                 title: Ts_4///geo_accession: GSM1079029///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: serous///characteristics_ch1.2: advanced disease stage: stage III///characteristics_ch1.3: recurrence status: recurrence///characteristics_ch1.4: survival/death: death///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: s4///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1079nnn/GSM1079029/GSM1079029_s4.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
                  title: Ts_5///geo_accession: GSM1079030///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: serous///characteristics_ch1.2: advanced disease stage: stage IV///characteristics_ch1.3: recurrence status: recurrence///characteristics_ch1.4: survival/death: death///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: s5///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1079nnn/GSM1079030/GSM1079030_s5.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 
              title: Ts_8///geo_accession: GSM1079031///status: Public on Jan 01 2014///submission_date: Feb 06 2013///last_update_date: Jan 01 2014///type: RNA///channel_count: 1///source_name_ch1: primary surgical treatment///organism_ch1: Homo sapiens///characteristics_ch1: tissue: ovarian cancer cells///characteristics_ch1.1: tissue subtype: serous///characteristics_ch1.2: advanced disease stage: stage III///characteristics_ch1.3: recurrence status: recurrence///characteristics_ch1.4: survival/death: survival///treatment_protocol_ch1: Frozen rissues///growth_protocol_ch1: Frozen tissue samples were embedded, and sections were cut and mounted on slides. The slides were stained with H&E to ensure that at least 70% of the cellular content was comprised of epithelial ovarian cancer cells.///molecule_ch1: total RNA///extract_protocol_ch1: Trizol extraction of total RNA was performed according to the manufacturer's instructions.///label_ch1: biotin///label_protocol_ch1: biotin///taxid_ch1: 9606///hyb_protocol: Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Human Genome U133 Plus 2.0 Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 450///scan_protocol: The arrays were scanned to a target intensity of 500 by a GeneArray Scanner, and patterns of hybridization were detected as light emitted from the fluorescent reporter groups incorporated into the target and hybridized to oligonucleotide probes.///description: s8///data_processing: The CEL files were generated using Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings. Signal intensities were derived from CEL files and Signal intensity derived from the CEL files, using GCRMA method.///platform_id: GPL570///contact_name: Tzu-Hao,,Chang///contact_department: Graduate Institute of Biomedical Informatics///contact_institute: Taipei Medical University///contact_address: 250 Wu-Xin Street///contact_city: Taipei///contact_zip.postal_code: 110///contact_country: Taiwan///supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/pub/geo/DATA/supplementary/samples/GSM1079nnn/GSM1079031/GSM1079031_s8.CEL.gz///data_row_count: 54675 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      1 

duplicates: 
   Length     Class      Mode 
       60 character character 

}}
\keyword{datasets}
